runtime/vm/xz64/UpcallThunkGen.cpp

```diff
@@ -0,0 +1,930 @@
+/*******************************************************************************
+ * Copyright (c) 2022, 2022 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available undeMAX_MVC_COPY_LENGT * Secondary Licenses when the conditions for such availability set
```

Comment:
Seems like some kind of mixed up here.

Reply:
Should be fixed here: 565c53270b39f2e5b6923add77381ea9975c9336